flowpm/tfpm.py

```diff
@@ -416,3 +590,96 @@ def nbody(cosmo,
       return intermediate_states
     else:
       return state
+
+
+def nbody_pgd(cosmo,
```

Comment:
Ohhh wait.... what is this function supposed to be doing exactly? It fits and apply pgd online???

Reply:
Yeah, this is supposed to apply PGD after the drift step...so small correction to positions of the particles.
Currently it's fitting for these PGD parameters at every redshift. But you do not want to be the case when you are running a simulation since there you will ideally be feeding in the fitted parameters. 
So we need a separate script/file for fitting PGD parameters and applying PGD.
I would say apply_PGD should still be in tfpm.py but not fit_pgd.

Reply:
cool cool thanks :-)